docs/source/asr/configs.rst

```diff
@@ -511,13 +511,22 @@ Conformer-Transducer
 
 Please refer to the model page of `Conformer-Transducer <./models.html#Conformer-Transducer>`__ for more information on this model.
 
+RNN-Transducer
+~~~~~~~~~~~~~~
+
+The config files for RNN-Transducer (RNNT) and RNN-CTC models can be found at ``<NeMo_git_root>/examples/asr/conf/rnn/rnn_transducer_bpe.yaml`` and ``<NeMo_git_root>/examples/asr/conf/rnn/rnn_ctc_bpe.yaml`` respectively.
```

Comment:
Hmm considering our over use of RNN transducer, can we rename this model to (LN)LSTM Transducer? Highly confusing that we use RNN-T, RNNT, Transducer etc to refer to Conformer ContextNet and now LSTM encoders.

Reply:
Strictly speaking, its not the basic RNN encoder but an Layer Norm LSTM encoder with a transducer decoder.

Reply:
changed names.